src/Quizzer.API.BL.UnitTests/QuestionFacadeTest.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System.Globalization;
+
+namespace Quizzer.API.BL.UnitTests {
+    public class QuestionFacadeTest {
+        private Mock<IRepository<QuestionEntity>> _questionRepositoryMock = new(MockBehavior.Strict);
+        private Mock<IUnitOfWork> _uowMock = new(MockBehavior.Strict);
+        private Mock<IUnitOfWorkFactory> _uowFactoryMock = new(MockBehavior.Strict);
+        private Mock<IMapper> _mapperMock = new(MockBehavior.Strict);
+
+        [Fact]
+        public void GetDetailById_Sets_Question_Types_Sets_Correctly_AnswerType() {
+            // Arrange
+            _questionRepositoryMock = new Mock<IRepository<QuestionEntity>>(MockBehavior.Loose);
+            QuestionFacade questionFacade = GetFacadeWithForbiddenDependencyCalls();
+
+            var questionDetailModel = new QuestionDetailModel {
+                Id = Guid.NewGuid(),
+                Text = "Name of quiz",
+                QuestionType = DAL.Enums.AnswerFormat.SingleChoice,
```

Comment:
It would be better to make `using` from the namespace

Reply:
I am not sure what exaclty you mean

Reply:
Add `using DAL.Enums;` and remove the full paths for class types.

Reply:
done